docs/tutorials/vibration_and_jittering.rst

```diff
@@ -0,0 +1,196 @@
+
+Analyzing Logs and Flight Controller Data
+*******************************************
+The best way to diagnose possible vibration problems is to analyze the logs produced by your flight controller. 
+Both PX4 and Ardupilot based flight controllers have tools available to help with reviewing their logs and identifying potential issues.
+
+PX4
+####
+`Flight Review <https://docs.px4.io/main/en/log/flight_review.html#log-analysis-using-flight-review>`_ is a very useful tool provided by PX4 that simplifies analyzing logs and identifying problems.
+
+First, configure the `logging profile <https://docs.px4.io/main/en/advanced_config/parameter_reference.html#SDLOG_PROFILE>`_ on your flight controller. The default logging set is a good starting point, 
+but we also recommend turning on `high-rate logging <https://docs.px4.io/main/en/advanced_config/parameter_reference.html#SDLOG_PROFILE>`_ before running any 
+tests to try and diagnose the issue. Having high-rate logging turned on increases the amount of data logged by the flight-controller and allows additional useful analysis features in 
+Flight Review, such as an FFT on the actuator output. 
+
+There is also the option to turn on `raw FIFO high-rate data from the IMU or gyroscope <https://docs.px4.io/main/en/log/flight_review.html#raw-high-rate-imu-data-plots>`_. This is generally not necessary for an initial diagnosis of vibration issues, but if you want to 
+look specifically at high frequencies it can be useful. This does add considerably to the logging bandwidth, so it can lead to drop-out of logging data and large log file sizes, 
+so it is best to avoid using those settings if it is not necessary.
+
+Once your logging profile is appropriately configured, run a ground or flight test to replicate the issue and generate a log. Then, upload the `log to Flight Review <https://logs.px4.io/>`_. 
+This should generate a series of plots summarizing the data from the log. 
+
+The PX4 wiki page on `Log Analysis using Flight Review <https://docs.px4.io/main/en/log/flight_review.html>`_ provides an overview of how to use these plots to analyze the performance of your vehicle. 
+We will focus on a few of the most relevant sections for analyzing vibration issues here.
+
+Raw sensor data can be a useful place to start when looking for vibration issues. Periods of extreme oscillation are a sign that vibrations are causing issues for the sensors. 
+The graph below shows a particularly bad example of what this can look like for raw acceleration.
+
+.. figure:: ../_static/tutorial_images/vibration_tutorial/bad_raw_sensor_example.png
+        :align: center
+        :width: 60%
+        :alt: Example of Raw Sensor Data Showing Impact of Vibrations
+
+        Example of Raw Sensor Data Showing Impact of Vibrations
+
+Flight Review also provides graphs of the FFT of sensor data if high-rate logging is turned on, which can help identify the specific frequencies of vibration and how severe they are. 
```

Comment:
I'm not going to use exactly the wording here, because you can have multiple frequencies of vibration. I'm going to go with something that pluralizes what you have here